tests/integ/modin/test_numpy.py

```diff
@@ -86,6 +86,17 @@ def test_full_like():
         pandas_result = np.full_like(pandas_df, "numpy is the best")
         assert_array_equal(np.array(snow_result), np.array(pandas_result))
 
+    with SqlCounter(query_count=1):
+        pandas_result = np.full_like(pandas_df, fill_value=4, shape=())
+        # breakpoint()
+        snow_result = np.full_like(snow_df, fill_value=4, shape=())
+        assert_array_equal(np.array(snow_result), np.array(pandas_result))
+
+    with SqlCounter(query_count=1):
+        snow_result = np.full_like(snow_df, fill_value=4, shape=(4))
```

Comment:
this is equivalent to `shape=4`. did you mean to test `shape=(4,)`?

Reply:
Added (4,); but I actually did want (4) in this case. So now I have both cases.

Reply:
I would delete the `(4)` test case since it means the same thing as `4` and it looks like it means `tuple(4)`.